File: p9.R
ptm <- proc.time()

for (a in 1:500) {
  for (b in (a+1):500) {
    c = sqrt((a^2) + (b^2))
    if (c%%1 == 0) {
      if ((a+b+c) == 1000) {
        print(sprintf("a: %.0f, b: %.0f, c: %.0f, abc: %.0f", a, b, c, (a*b*c)))
      }
    }
  }
}

print(proc.time() - ptm)

File: p9.py
from math import sqrt

for a in range(1, 501):
	for b in range(a+1, 501):
		c = sqrt((a**2) + (b**2))
		if int(c) == c:
			if (a+b+c) == 1000:
				print "a: %d, b: %d, c: %d, abc: %d" % (a, b, c, (a*b*c))